forge-gui/res/cardsfolder/upcoming/unleash_the_inferno.txt

```diff
@@ -0,0 +1,9 @@
+Name:Unleash the Inferno
+ManaCost:1 B R G
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | NumDmg$ 7 | ExcessSVar$ Excess | SubAbility$ DBImmediateTrigger | SpellDescription$ CARDNAME deals 7 damage to target creature or planeswalker. When it deals excess damage this way, destroy target artifact or enchantment an opponent controls with mana value less than or equal to that amount of excess damage.
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionCheckSVar$ Excess | ConditionSVarCompare$ GE1 | RememberSVarAmount$ Excess | Execute$ TrigDestroy | TriggerDescription$ When it deals excess damage this way, destroy target artifact or enchantment an opponent controls with mana value less than or equal to that amount of excess damage.
+SVar:TrigDestroy:DB$ Destroy | ValidTgts$ Artifact.OppCtrl+cmcLEX,Enchantment.OppCtrl+cmcLEX | TgtPrompt$ Select target Artifact or Enchantment | SubAbility$ DBCleanup

```

Comment:
TgtPrompt$ Select target artifact or enchantment an opponent controls with mana value less than or equal to the amount of excess damage